implementations/py/README.md

```diff
@@ -0,0 +1,40 @@
+# polywrap-ethereum-plugin
+The Ethereum Provider plugin implements the `ethereum-provider-interface` @ [ens/wraps.eth:ethereum-provider@2.0.0](https://app.ens.domains/name/wraps.eth/details) (see [../../interface/polywrap.graphql](../../interface/polywrap.graphql)). It handles Ethereum wallet transaction signatures and sends JSON RPC requests for the Ethereum wrapper.
+
+## Usage
+### 1. Configure Client
+When creating your Polywrap JS client, add the ethereum provider plugin:
+```python
+from polywrap_client import PolywrapClient
+from polywrap_ethereum_provider import ethereum_provider_plugin
+
+ethereum_provider_plugin_uri = Uri.from_str("plugin/ethereum-provider")
+connections = Connections(
+    connections={
+        "mocknet": Connection(provider, None),
+        "sepolia": Connection.from_network(KnownNetwork.sepolia, None)
```

Comment:
this makes me think, we're passing `signer` in `Connection` and `Connections`; do we want this?

Reply:
Do you mean, why do we allow passing signers in both connection and connections?

Reply:
It's just UX, the user may want to use the same key for every connection or may want separate keys for every connection.